src/Init/WF.lean

```diff
@@ -268,7 +268,7 @@ section
 variable {α : Type u} {β : Type v}
 variable {ra  : α → α → Prop} {rb  : β → β → Prop}
 
-def lexAccessible {a : α} (aca : Acc ra a) (acb : (b : β) → Acc rb b) (b : β) : Acc (Prod.Lex ra rb) (a, b) := by
```

Comment:
Isn't there some reduction-related reason that `Acc` results should be `def`s`?

Reply:
Yes. Several definitions here need to reduce so that well founded recursive definitions will compute. (I discovered these are load-bearing when working on lean4lean, it is reduced inside lean core itself.)

Reply:
So does this change need to be reverted, or are these not such definitions?

Reply:
None of the definitions marked as `@[reducible]` were modified, so it seems fine to me